mmedit/engine/optimizers/optimizer_constructor.py

```diff
@@ -1,5 +1,9 @@
 # Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
decompose to singanoptimwrapperconstructor.py and pgganoptimwrapperconstrucotor.py